src/css/components/modal.css

```diff
@@ -0,0 +1,71 @@
+.display-none {
+  display: none;
+}
+.display-block {
+  display: block;
+}
+.modal {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  background-color: #00000059;
+  z-index: 99999;
```

Comment:
`z-index` values are relative to the [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) of the elements, so `1` probably would work here just as well as `9999` 🙂